app/search_builders/collection_search_builder.rb

```diff
@@ -9,9 +11,9 @@ def include_item_ids(solr_parameters)
     solr_parameters[:fq] << "hasCollectionMember_ssim:#{item.id}"
   end
 
-  def initialize(item, processor_chain, scope)
```

Comment:
It's not clear to me why `#initialize` is going away in favor of `#include_contained_files`. Could you say a bit about this change?